pydal/adapters/base.py

```diff
@@ -1342,7 +1342,10 @@ def represent(self, obj, fieldtype):
             elif not isinstance(obj, (list, tuple)):
                 obj = [obj]
             if field_is_type('list:string'):
-                obj = map(str,obj)
+                try:
+                    obj = map(str,obj)
+                except UnicodeEncodeError:
+                    obj = map(lambbda x:unicode(x).encode(self.db_codec),obj)
```

Comment:
Is there a way to resubmit a pull with the error fixed or do I just keep making new ones?